__tests__/components/editor/Editor.test.js

```diff
@@ -22,16 +22,24 @@ describe('<Editor />', () => {
     expect(wrapper.find(ResourceTemplate).length).toBe(1)
   })
 
-  it('renders <StartingPoints /> component', () => {
-    expect(wrapper.find(StartingPoints).length).toBe(1)
-  })
-
```

Comment:
it's fine that StartingPoints isn't here ... but will there still be a list of resource templates for users somewhere?  Is hooking up that work with this work already captured in an issue?

Reply:
Yes, see this design from @astridu https://projects.invisionapp.com/share/WDQFD026J9F#/screens/345867056_Profiles_Home on the Import Resource Template tab. I think this display is a stretch goal for this sprint.